client/client.go

```diff
@@ -256,6 +256,7 @@ func (c httpMetricClient) retryWithBackoff(
 		var uploadURL, hash string
 		uploadURL, hash, err = c.GetUploadURL(metricFile, metricSampleURL, agentVersion, UID)
 		if err != nil {
+			log.Debugf("Client proxy or deployment YAML may be misconfigured.  Please check your client settings.")
```

Comment:
Will we get Debug logs from the clients? Is that level enabled by default on their deployments or is that something we'd have to ask them to change to be able to get this additional info?

Reply:
I see it's done elsewhere in the code, so maybe it is on by default...nbd if you're not sure, I'm sure it's fine. At this point I'm just curious if you already know how we handle debug level logging in the metrics agent.

Reply:
They'd have to enable it, but they can do it in the deployment and then let the pod restart and see this error.  The Readme mentions it as one of the configuration options.